######################################
########### Param for SRIBO ##########
######################################

delta: 3                  # 
lopt: 10                  #滑窗初始长度
k: 1                      #
rate: 25                  #hz
dt: 0.04                  #dt
paramP: 0.1               #P矩阵系数 - x0
paramQ: 1                 #Q矩阵系数 - imu
paramR: 1                 #R矩阵系数 - flow 1
enable_flow: false        # true / false
order: 3                  #多项式阶数

# --- param aerial drags for axis x, y, z --- #

# param used for SRIFO in data set F0
kappa: 
      [0.05,
       0.5,
       1.5 ]

# param used for SRIO in data set F0
# kappa: 
#       [1.8,
#        0.3,
#        0.6 ]
